CONTRIBUTING.md

```diff
@@ -1,71 +1,112 @@
-        * All source and test files include a license header.
-        * The [`LICENSE`](LICENSE) file is included in your root directory.
-
-* Tests that are located in `/tests/` and discoverable through `python -m unittest discover`.
-
-Our examples are tested using CircleCI. For a list of operating systems and
+1.  Follow GitHub's instructions for
+    [creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
+    Alternatively, clone this repository as a starting point from which to implement your example.
+
+2.  Implement your example by following these instructions:
+
+    1.  It is preferable that all Python and CSS code occur either in new files or in files that
+        contain `demo` in the name. Below is a list of `demo` files and their functionality:
+
+        *   [demo_callbacks.py](demo_callbacks.py) contains all the
+            [Dash callback functions](https://dash.plotly.com/basic-callbacks) required to run the
+            Dash app. Updates to existing callbacks and new callbacks should be made in this file.
+        *   [demo_interface.py](demo_interface.py) contains all the Dash HTML components that
+            generate the user interface (UI) for the app, including settings like sliders,
+            checkboxes, and text inputs, as well as buttons and tables. If a Dash HTML component
+            needs to be dynamically added to the UI by a Dash callback in `demo_callbacks.py`, it
+            is encouraged to add a function generating this component to `demo_interface.py` and
+            call this function from `demo_callbacks.py`.
+        *   [demo_configs.py](demo_configs.py) contains all configurations and settings for the
+            demo and is intended as a quick way to customize the demo for a specific audience or
+            use case. It is encouraged to add more customization settings to this file; however,
+            this file should not be cluttered with settings that are not helpful to the end user
+            (eg titles and other settings that are unlikely to ever be changed).
+        *   [demo.css](demo.css) contains all custom CSS styling for the demo; any new CSS rules
+            should be added here or in a new file in the `/assets/` directory. Dash reads all
+            files in `/assets/` in alphabetical order so, out of the provided CSS files, `demo.css`
+            runs last and can overwrite previous styling rules.
+        *   [__demo_variables.css](__demo_variables.css) contains all CSS variables.
+            It is encouraged to edit or add more CSS variables here.
+        *   [demo_enums.py](demo_enums.py) contains [Enum](https://docs.python.org/3/library/enum.html)
+            classes for any settings or variables are being used frequently to avoid string
+            comparisons or other fragile code practices.
```

Comment:
```suggestion
        *   [demo_enums.py](demo_enums.py) contains [Enum](https://docs.python.org/3/library/enum.html)
            classes for any settings or variables that are being used frequently to avoid string
            comparisons or other fragile code practices.
```